core/modules/layout/tests/layout.test

```diff
@@ -1133,14 +1133,22 @@ class LayoutInterfaceTest extends BackdropWebTestCase {
     );
     $this->backdropPost('admin/structure/layouts/manage/' . $layout_name . '/configure', $edit, t('Save layout'));
 
-    $deleted_uuids = $default_layout->positions['sidebar'];
+    $orphaned_uuids = $default_layout->positions['sidebar'];
     layout_reset_caches();
     $layout = layout_load($layout_name);
-    $this->assertBlocksMatch($layout, $default_layout);
-    $this->assertFalse(isset($layout->positions['sidebar']), 'Sidebar region blocks removed from layout when changing layout to single column.');
+    $this->assertBlocksMatch($layout, $default_layout, $orphaned_uuids);
```

Comment:
Although admittedly highly unlikely with tests, is changing function parameters an API change? We could set a `$orphaned_uuids` property on the test class instead and check for that in `assertBlocksMatch()`
```
    $this->orphaned_uuids = TRUE;
    $this->assertBlocksMatch($layout, $default_layout);
```

Reply:
assertBlocksMatch is only being used in this class and actually only in the one method. It's a protected method too so it's not really an API in my opinion.